HIP/hip-991.md

```diff
@@ -123,7 +108,7 @@ We propose adding a fixed fee mechanism to the Hedera Consensus Service (HCS) fo
   
 ##### Handling Duplicates
 
-If the FEKL list contains duplicate keys, the transaction will fail with an `FEKL_CONTAINS_DUPLICATED_KEYS` error. This ensures that duplicate entries are not silently ignored, preventing potential bugs or issues in the calling code.
+If the FEKL list contains duplicate keys, the `ConsensusCreateTopicTransaction`, respectively `ConsensusUpdateTopicTransaction`, will fail with a `FEKL_CONTAINS_DUPLICATED_KEYS` error. This ensures that duplicate entries are not silently ignored, preventing potential bugs or issues in the calling code.
```

Comment:
The `FEKL_CONTAINS_DUPLICATED_KEYS` can be renamed to `FEE_EXEMPT_KEY_LIST_CONTAINS_DUPLICATED_KEYS`.

Reply:
Good idea. Changed it.